src/Components/Server/src/DependencyInjection/ConfigureStaticFilesOptions.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.StaticFiles;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNetCore.Components.Server.DependencyInjection
+{
+    public class ConfigureStaticFilesOptions : IPostConfigureOptions<StaticFileOptions>
```

Comment:
Can this be internal? Also don't use that namespace. Either `Microsoft.Extensions.DependencyInjection` or `Microsoft.AspNetCore.Components.Server` is fine.

Reply:
Ah, yeah, my bad

Reply:
Done